### DISCUSSION TITLE
Play Audio On Camera When Motion Detected

### DISCUSSION THREAD

**Discussion by ryancurrah** [blakeblackshear/frigate#14509] [Ask A Question] [answered]
Is it possible to play an audio recording when the camera detects motion? I want to play a message to warn people at night they are trespassing. I could not find an answer to this in Github Issues/Discussions or the docs.

**Comment by NickM-27** [comment, marked as answer]
you can do this using an automation platform like home assistant or NodeRED

**Reply by ryancurrah** [reply]
Thanks for the quick and helpful response. So Frigate NVR has a way to launch a webhook on motion detection? (I am a new user of Frigate so not familiar with it entirely yet)

**Reply by NickM-27** [reply]
This would be done using MQTT as the trigger

**Reply by ryancurrah** [reply]
Ack that makes sense. I will create something and post my solution here when I got it working.
